app/models/user.rb

```diff
@@ -51,6 +51,7 @@ class User < ActiveRecord::Base
 
   before_validation :default_display_name, on: [:create, :update]
   before_validation :setup_unsubscribe_token, on: [:create]
+  before_validation :update_ouroboros_created, on: [:create]
```

Comment:
yeah - it looks like all the devise `save` methods are `validate: false`. As long as the before validate is working i'm good.